internal/lookoutingesterv2/ingester.go

```diff
@@ -0,0 +1,50 @@
+package lookoutingesterv2
+
+import (
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/G-Research/armada/internal/common/app"
+	"github.com/G-Research/armada/internal/common/compress"
+	"github.com/G-Research/armada/internal/common/database"
+	"github.com/G-Research/armada/internal/common/ingest"
+	"github.com/G-Research/armada/internal/lookoutingesterv2/configuration"
+	"github.com/G-Research/armada/internal/lookoutingesterv2/instructions"
+	"github.com/G-Research/armada/internal/lookoutingesterv2/lookoutdb"
+	"github.com/G-Research/armada/internal/lookoutingesterv2/metrics"
+	"github.com/G-Research/armada/internal/lookoutingesterv2/model"
+)
+
+// Run will create a pipeline that will take Armada event messages from Pulsar and update the
+// Lookout database accordingly.  This pipeline will run until a SIGTERM is received
+func Run(config *configuration.LookoutIngesterV2Configuration) {
+	log.Infof("Opening connection pool to postgres")
+	m := metrics.Get()
+	db, err := database.OpenPgxPool(config.Postgres)
+	if err != nil {
+		panic(errors.WithMessage(err, "Error opening connection to postgres"))
+	}
+	lookoutDb := lookoutdb.NewLookoutDb(db, m, 10, 60)
```

Comment:
let's comment what the magic numbers are here!

Reply:
Extracted into config and commented there, think it's sensible